src/main/java/org/entando/plugins/pda/core/engine/Connection.java

```diff
@@ -3,29 +3,27 @@
 import java.util.Map;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
-import lombok.ToString;
 
-@Getter
```

Comment:
Connection não deveria ter `@Data` pois não serializado/deserializado.
`@Getter @Setter` é mais correto.

Reply:
`@Data` has nothing to do with serialization/deserialization requirements. According to its Javadoc:
```
Generates getters for all fields, a useful toString method, and hashCode and equals implementations that check all non-transient fields. Will also generate setters for all non-final fields, as well as a constructor.
Equivalent to {@code @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode}.
```
So, every time you have all those annotations together, you can and you should replace for `@Data`. More details here: https://projectlombok.org/features/Data

Reply:
Cool, nice to know this!